services/QuillLMS/engines/evidence/lib/evidence/evidence/plagiarism_check.rb

```diff
@@ -124,13 +124,20 @@ def feedback_object
     end
 
     private def identify_matched_slices(entry_slices, passage_slices)
```

Comment:
Because we use the `identify_matched_slices` function to generate highlights in two different directions (one where we pass `(passage_slices, entry_slices)` and one where we pass `(entry_slices, passage_slices)`, each being used to generate separate highlights (one for the entry, and one for the passage), it probably makes sense to tweak the names of the input params to be less prescriptive.  I'll do that.